### DISCUSSION TITLE
Option to disable wrapping <article> tag

### DISCUSSION THREAD

**Discussion by n-older** [markdoc/markdoc#324] [Q&A] [answered]
It would be nice if there was an option to disable the enclosing `<article></article>` tag.  While I understand this is a great default for most users, in some cases like mine it can have adverse effects.

For example, we are using Sanity.io as a CMS that has both a `title` and `body` field and is run through a static site generator. The title field ultimately sets the article's page title and `<h1>` while the body field is rendered into a child `<div>` of an existing `<article>` tag, which results in a duplicate, nested `<article>` tag.

For now I am just using regex to remove the enclosing `<article>` tag before it is rendered but this seems like a good thing to be able to configure within Markdoc.

**Comment by mfix-stripe** [comment, marked as answer]
@nolder3193 you can disable or customize this behavior by defining a custom `document` [node](https://markdoc.dev/docs/nodes).

See this source for how the default document node is defined: https://github.com/markdoc/markdoc/blob/main/src/schema.ts#L5

**Reply by n-older** [reply]
Ah okay, I was searching for "article" in the docs and nothing came up. My mistake.

Thanks!

**Reply by mfix-stripe** [reply]
Not your mistake at all! We should do a better job at documenting these concepts.
